### PR TITLE
Fixes #9890 - Add content_view_id to systems API

### PR DESCRIPTION
**Pull request by daviddavis**

### CODE REVIEW

File: app/controllers/katello/api/v2/systems_controller.rb [hunk 1]
```diff
@@ -61,6 +61,7 @@ def organization_id_keys
     param :organization_id, :number, :desc => N_("Specify the organization"), :required => true
     param :environment_id, String, :desc => N_("Filter by environment")
     param :host_collection_id, String, :desc => N_("Filter by host collection")
+    param :content_view_id, String, :desc => N_("Filter by content_ivew")
```
Comment (cfouant) [inline app/controllers/katello/api/v2/systems_controller.rb]: Can you fix the description? I think it should just say "Filter by content view"

**Comment by cfouant** [comment]
The filtering works for me. ACK from me pending the changes requested above!

File: app/controllers/katello/api/v2/systems_controller.rb (hunk 1 above)
Comment (daviddavis) [inline app/controllers/katello/api/v2/systems_controller.rb]: :+1:

**Comment by daviddavis** [comment]
[test]

**Pull request merged**